### ISSUE TITLE
May Project Update to ROW Management 

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#160]

**Comment by TracyLinder** [comment]
Sent today:

Hi ROW Managers,
 
Ryan Ward asked me to send a project update on the various iron in the fires. This is long overdue, and I'm going to put a monthly task on my calendar to send an update to you all at the end of each month.

There are a few acronyms below, but I think the only one you may not be familiar with is UAT (User Acceptance Testing). Please let me know if you have any questions or concerns with any of the information provided here. Big things are coming, and June is shaping up to be a big month! 

PS – I’ll be on a road trip from 6/24 – 7/8. I’m wondering if we should hold off on going live with post payment, click map, and the PSM folder until I get back. Thoughts? 

Thanks,
Tracy
 
ROWMAN Projects

Phase 1 (Closed) - Went live December 2018
Deliverables: Migration to AB&C Portal, new applications went online. 
 
Phase 2 (Active) - Click Map and PSM Folder
Deliverables:
•	Basic click map will be added to ROW permit applications on AB&C portal, allowing applicants the ability to use a map to add streets to the application. 
o	Status: COA GIS team is fixing some bugs found last week. Expected to go live mid-June. 
•	Advanced click map will allow applicants to choose parking spaces from the map, import CIP projects, import & edit previous ROW applications, will auto-populate certain AMANDA folder infos like ROW Inspector and protected street status, and will import parking related data to the AMANDA folder like meter number, zone type, parking benefit district, and more. 
o	Status: Waiting on ATD parking data. Then Unisys will have some development work to do on the parking free form tab. UAT projected to start in August or September. 
•	New PSM folder for temp sidewalk cafes. Applicants will be able to apply online. 
o	Status: In UAT. Depending on UAT results, this could go-live mid-June. 
 
Phase 3 (Active) - Post Payment & Contractor Self-Assignment
Deliverables:
•	Post Payment will automate fees on almost all ROW permits. 
o	Status: Pending bug fixes from Unisys. Projected final round of UAT will begin the week of June 3rd. If UAT goes well, this will go live mid-June. 
•	UC & LM Self Assignment will allow UC & LM applicants to submit applications for permit folders auto-created during the SP review. 
o	Status: The first round of testing should begin the 1st week of June. I will need testing resources from the UC team. Also testing are Greg Pepper, and Andy Halm. 
•	EX & RW Self Assignment - On hold until phase 4 is closed. 
 
Phase 4 (ROWAC) – Road Closure Notifications & Real-Time Closure Data
Deliverables: 
•	Upgrade & updates to the current 311 notification process, and new features to allow for collecting Real-Time Closure Data. Add a real-time closure layer to ROWPACT.  
o	Status: I will start working on a proof of concept to present to this group after PSM UAT is complete, and I will request approval (from you) to move forward with development. The ROWMAN Project will be out of funds at this point. I'll work with Unisys to get an estimate for phase 4, then ask the CMGB for additional funding. Whatever cost is not covered by the CMGB will be requested from ROW. Once funds are secured, I will have to go back to the CMGB and request spending authority for the engagement with Unisys. (I'll be meeting with the ROWMAN project manager next week to finalize this plan.) I've conveyed that this is our highest priority, so as soon as I get approval on the proof, the project team is ready to work as quickly as possible on getting through the funding process and starting development. The goal is to start development in August. 
 
Phase 5 & After - Deliverables unclear. We will revisit the continuation of this project as we near the end of phase 4. 
 
ROW Audit Deliverables Project
I spoke to Stacey Wuest early this month about getting development resources from her team assigned to this project, and she is unable to do so any time soon. She suggested we wrap this up into the ATD AMANDA Backlog Project. 
 
ATD AMANDA Backlog Project
This became an official project earlier this month. 
Deliverables: 
o	Development of everything in ATD's AMANDA backlog, including the audit deliverables. 
o	Status: Check this out. It's got links to everything that needs to be done so development on this can start. I'm currently cleaning up the user stories so I can get an estimate from Unisys. Ryan Mooney - you'll need to talk this up at the next CMGB meeting, and get them prepared to approve funding for it at the following meeting. 
 
Inspector Priority Dashboard
Deliverables: 
o	Tools/lists to help the ROW inspectors better prioritize their work. 
o	Phase 1 Status: John gave me a markup to review yesterday. I hope to have this reviewed today, and will schedule a meeting with the team to review and plan next steps. 

Land Management Portal Applications
•	This is not an ATD project, but the application link will be nested under the ‘Apply for ROW Permits’ link on the portal. I’ve asked the Office of Real Estate Management to assign resources to Fresh Chat, so they can assist online customers with LM chats. 
o	Status: Unsure. They will give me a 1 month notice before they go live, so I can help them get setup on Fresh Chat. ROW Fresh Chat users will just need to be aware of how to redirect LM customers when the time comes. 

Tracy Linder
IT Business Systems Analyst Sr.
Data and Technology Services 
Austin Transportation Department
